docs/components/Date.md

```diff
@@ -0,0 +1,87 @@
+    - **Default:** True
+- **Submit enabled - SubmitEnabled** - Enables submit functionality. When this is false, live validation is used. Need to hook up component to a button control to trigger validation with submit functionality.
+    - **Default:** True
+- **Submit - Submit** - Boolean to trigger the on submit functionality. Only works when SubmitEnabled is set to true.
+    - **Default:** False
+- **Theme - Theme** - Configures the styling of component elements.
+    - **Default:** See [Theme Documentation](../Theme.md).
+- **Show day - ShowDay** - Controls the visibility of the day field.
+    - **Default:** True
+- **Show year - ShowYear** - Controls the visibility of the year field.
+    - **Default:** True
+- **Default day - DefaultDay** - Day to be used in the date output value when the day field is not visible.
+    - **Default:** 01
+- **Default year - DefaultYear** - Year to be used in the date output value when the year field is not visible.
+    - **Default:** 1972
+- **Max date - MaxDate** - Maximum date that will be accepted as an input into the component.
+    - **Default:** DateValue("31/12/9999")
+        - This is the limit of the maximum date input property, so any date set should not exceed this default.
+- **Min date - MinDate** - Minimum date that will be accepted as an input into the component.
+    - **Default:** DateValue("01/01/1900")
+        - This is the limit of the minimum date input property, so any date set should not precede this default.
+
+### Outputs
+
+- **Value - Value** - Outputs the sanitised date input.
+- **Date format - DateFormat** - Outputs common date formats, based on the fields visible in the component.
+    - If all fields are visible, the date format output is "dd/mm/yyyy".
+    - If the day and month fields are visible but the year field is not visible, the date format output is "dd/mmm".
+    - If the month and year fields are visible but the day field is not visible, the date format output is "mmm/yyyy".
+- **Has error - HasError** - Boolean value based on whether an error is present in the component.
+- **Min height - MinHeight** - Minimum height for all visible elements of the component to be shown. This is responsive to the height of the component, so no part of the component is ever cut off. This is the default value of the component's height property.
+
+### Output Sanitisation
+
+The output for the date component (Date.Value) is a DateValue comprising of the day, month and year inputs. There are exceptions to this, where the fields are not visible in the component.
+- If the year field is not visible (i.e. ShowYear is false), then the DateValue will use the default year set as an input property.
+- If the day field is not visible (i.e. ShowDay is false), then the DateValue will use the default day set as an input property.
+
+Note, the output sanitisation uses the Date formula. This means when a year is input that is less than 1900, the year output will be the year input + 1900.
```

Comment:
Is there a link to any official documentation describing this behaviour? Would be good to link that here